pupil-spa/src/app/question/question.component.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('QuestionComponent', () => {
     component = fixture.componentInstance;
     fixture.detectChanges();
     // prevent Timeout's being set
-    spyOn(component, 'ngAfterViewInit').and.returnValue(null);
+    //spyOn(component, 'ngAfterViewInit').and.returnValue(null);
```

Comment:
@jon-shipley commenting this out and then explicitly calling `component.ngAfterViewInit()` within the test is the only way i have managed to make the test pass.